crates/cast/src/lib.rs

```diff
@@ -350,6 +350,27 @@ pub fn print_formatted(output: Vec<(&str, String)>, json: bool, error: bool) ->
     Ok(())
 }
 
+fn json_value_to_string(value: &Value, value_format: ValueFormat) -> Option<String> {
+    match value {
+        Value::Number(n) => {
+            let n = n
+                .as_u64()
+                .unwrap_or_else(|| panic!("failed to convert {n} to u64"));
```

Comment:
let's tru to avoid panics :+1:

Reply:
it's essentially the same as `.expect`
https://stackoverflow.com/questions/55513198/is-there-a-more-concise-way-to-format-expect-message